src/components/panel/panel.js

```diff
@@ -923,9 +1043,23 @@ MdPanelRef.prototype.open = function() {
   return this._$q(function(resolve, reject) {
     var done = self._done(resolve, self);
     var show = self._simpleBind(self.show, self);
+    var setOpened = function(response) {
+      self.isOpened = true;
+      return response;
+    };
+    var checkGroupMaxOpen = function(response) {
```

Comment:
Another option is to make `checkGroupMaxOpen` pass when there isn't a group, which simplifies this method.